float32 northing_min
float32 northing_max
float32 easting_min
float32 easting_max